sw/device/silicon_creator/lib/ownership/isfb.c

```diff
@@ -133,7 +133,7 @@ rom_error_t isfb_info_flash_erase_policy_get(
 
   uint32_t check_cnt_exp = 0;
   uint32_t conditions = owner_config->isfb->erase_conditions;
-  hardened_bool_t policies[3];
+  hardened_bool_t policies[3] = {0};
```

Comment:
Explicitly initialize the policies array to zero.

Reply:
LGTM